proposals/repo-to-tar/README.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Building tarred bundles without being dependent on a OCI registry"
+authors: [ "Ashish Kumar <ashishndiitr@gmail.com>" ]
+status: "draft"
+approvers: [ ]
+---
+
+# <Building tarred bundles without being dependent on a OCI registry>
+
+## Problem Statement
+
+Presently imgpkg does not support any way of creating and storing bundles in any format directly to local disk for sharing purposes, so as to remove the need of a registry in between. This proposal aims to add this functionality to imgpkg.
+
+Consider this workflow, where a user wants to create a tar file of a bundle image and then push it to a registry. <br>
+In order to obtain a tar file, one needs to first push the bundle image to the registry and then leverage the [command](https://carvel.dev/imgpkg/docs/v0.37.x/air-gapped-workflow/#option-2-with-intermediate-tarball) for copying as tar in the air gapped workflow.
+
+For example, In order to obtain a tar of bundle image of examples/basic-step-2, we need run the following commands in succession :<br>
+
+`imgpkg push -b index.docker.io/user1/simple-app-bundle:v1.0.0 -f examples/basic-step-2` <br>
+`imgpkg copy -b index.docker.io/user1/simple-app-bundle:v1.0.0 --to-tar /tmp/my-image.tar`
+
+This has 3 fold issues : <br>
+1. Dependent on using a registry to create a tar file from a bundle image.
+2. The tar file is created from the bundle image, which is not required if the user just wants to create a tar file having a layer as tar with configurations/metadata.
+3. The bundle or image and it's tar file formed is not OCI compliant. This proposal aims to make the tar in a oci spec compliant format.
+
+## Terminology / Concepts
+1. Definitions of terms used in the proposal with respect to imgpkg can be found [here.](https://carvel.dev/imgpkg/docs/v0.37.x/)<br>
+2. Resources regarding docker save and docker load can be found [here.](https://docs.docker.com/engine/reference/commandline/save/)<br>
+3. A good article to understand OCI Artifacts can be found [here.](https://dlorenc.medium.com/oci-artifacts-explained-8f4a77945c13)
+4. Read about oci-spec in detail [here.](https://github.com/opencontainers/image-spec/blob/main/image-layout.md)
+
+## Proposal
+
+#### imgpkg
+
+Oci Tar Creation : <br>
```

Comment:
If we improve the push command, there is no need to add an extra command that will only do the save folders to the OCI tar format.

I suggest we only add the flag `--to-oci-tar` in the push command.

Reply:
Removed the save command from the proposal.